### PR TITLE
string/endsWith

### LINKED ISSUE #175: string/endsWith

### PR DESCRIPTION
**Pull request by bookercodes**
Closes #175.

### CODE REVIEW

File: test/string/endsWith.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+describe('endsWith', function () {
+  it('returns true if input ends with given string', function () {
+    expect(endsWith('foo', 'oo')).to.be.true;
+    expect(endsWith('bar', 'ar')).to.be.true;
+  });
+
+  it('returns false if input does not end with given string', function () {
+    expect(endsWith('foo', 'fo')).to.be.false;
+    expect(endsWith('bar', 'ba')).to.be.false;
+    // expect(endsWith('cake', 'k', 1)).to.be.false;
```
Comment (EvanHahn) [inline test/string/endsWith.js]: Could you remove this comment?

File: string/endswith.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@returns {bool} Does the string end with another string?
+@example
+var endsWith = require('lildash/string/endsWith');
+endsWith('food', 'ood');   // => true
+endsWith('cake', 'ood');   // => false
+endsWith('cake', 'k');     // => false
+endsWith('cake', 'k', 3);  // => true
+*/
+module.exports = function (str, query, position) {
+  position = position || str.length;
```
Comment (EvanHahn) [inline string/endswith.js:17]: What happens if `position === 0`?

File: string/endswith.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
+Does a string end with another string?
+
+@module string/endsWith
+@param {string} str The string to search through.
+@param {string} query The string to find in `str`.
+@param {number} [position=0] The index to end the search. Defaults to the end of the string.
```
Comment (EvanHahn) [inline string/endswith.js:7]: Is this default correct?

File: string/endswith.js (hunk 2 above)
Comment (bookercodes) [inline string/endswith.js:17]: The same thing that happens when `position` is [_falsey_](https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript):

``` javascript
endsWith('bar', 'r') // => true
endsWith('bar', 'r', 0) // => true
```

File: string/endswith.js (hunk 3 above)
Comment (bookercodes) [inline string/endswith.js:7]: I think so, yes.

If `position` is not specified, it will default to `0`.

File: test/string/endsWith.js (hunk 1 above)
Comment (bookercodes) [inline test/string/endsWith.js]: Done.

**Comment by bookercodes** [comment]
@EvanHahn,

Is there anything else I can do for this pr?

**Comment by EvanHahn** [comment]
I'll take a look at this soon and get back to you.

**Comment by bookercodes** [comment]
@EvanHahn Alright, thanks.

**Comment by EvanHahn** [comment]
Thanks!

**Pull request merged**